src/main/resources/application.yaml

```
@@ -0,0 +1,40 @@
+server:
+  # TODO: change to your application's port number
+  port: 4550
+
+spring:
+  application:
+    name: HMCTS Recipes Backend
+  config:
+    import: "optional:configtree:/mnt/secrets/plumsi/,optional:configtree:/mnt/secrets/crumblesi/,optional:configtree:/mnt/secrets/toffeesi/"
```

```suggestion
    import: "optional:configtree:/mnt/secrets/plumsi/,optional:configtree:/mnt/secrets/crumblesi/,optional:configtree:/mnt/secrets/toffeesi/"
```
One more thing to be careful of, looks like it uses both but maybe one KV is better?

---

Probably same deal as other comments, but worth pointing out another place this is mentioned

---

These will be clean up when the shared infra consolidation is complete and a new generic KV is created.